pipelines/LSSTComCam/quickLook.yaml

```diff
@@ -8,55 +8,38 @@ imports:
       - fgcmBuildFromIsolatedStars
       - fgcmFitCycle
       - fgcmOutputProducts
+      # We exclude calibrateImage here in order to restore the performance-
+      # oriented task default configs.
+      - calibrateImage
 tasks:
   isr:
     class: lsst.ip.isr.IsrTaskLSST
     config:
       # Turn off slow steps in ISR
       doBrighterFatter: false
-  # TO DO: Replace with calibrateImage in a way that we can use the
-  # preSources downstream.
-  characterizeImage:
-    class: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
+  calibrateImage:
+    class: lsst.pipe.tasks.calibrateImage.CalibrateImageTask
     config:
-      doApCorr: false
-      doDeblend: false
-  calibrate:
-    class: lsst.pipe.tasks.calibrate.CalibrateTask
-    config:
-      requirePhotoCal: false
-      requireAstrometry: false
-      python: |
-          config.measurement.plugins.names = ['base_CircularApertureFlux',
-                                              'base_PsfFlux',
-                                              'base_LocalBackground',
-                                              'base_SdssCentroid',
-                                              'base_SdssShape',
-                                              'base_Variance',
-                                              'base_Jacobian',
-                                              'base_PixelFlags',
-                                              'base_GaussianFlux',
-                                              'base_SkyCoord',
-                                              'base_FPPosition',
-                                              'base_ClassificationSizeExtendedness',
-                                              'base_CompensatedTophatFlux',
-                                              ]
-          config.measurement.slots.shape = "base_SdssShape"
-          config.measurement.slots.psfShape = "base_SdssShape_psf"
-          config.measurement.plugins['base_PixelFlags'].masksFpAnywhere.remove('STREAK')
-          config.measurement.plugins['base_PixelFlags'].masksFpCenter.remove('STREAK')
-          config.doApCorr = False
-          config.doDeblend = False
-          config.astrometry.sourceSelector["science"].doRequirePrimary = False
-          config.astrometry.sourceSelector["science"].doIsolated = False
-      # TO DO: DM-41189 Get pipeline that will attach an atool here from lauren
```

Comment:
I assume these configs here are very similar to the new calibrateImage defaults?

Reply:
Yes, because those are effectively the AP configurations.